tests/Exports_Tests.cpp

```diff
@@ -244,7 +244,7 @@ TEST_CLASS(Exports_Tests)
         Assert::IsFalse(mockUserContext.IsLoggedIn());
         Assert::AreEqual(std::string(""), mockUserContext.GetUsername());
         Assert::AreEqual(std::string(""), mockUserContext.GetApiToken());
-        Assert::AreEqual(0U, mockUserContext.GetScore());
+        Assert::AreEqual({0U}, mockUserContext.GetScore());
```

Comment:
why is this necessary?

Reply:
To preserve the Win32 version. GetScore return size_t, but U is a literal for unsigned int, there exists no literal for size_t. If the braces aren't there it won't compile.

Reply:
`GetScore` returns `unsigned int`, and I don't see anything in this PR that changes that.